students/shodges/lesson08/circle.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import math
+
+class Circle(object):
+    _radius = None
+    _diameter = None
```

Comment:
these should not be class attributes

Reply:
@natasha-aleksandrova Fixed, thanks.  Submitted [PR 419](https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/pull/419).